materialize-css/materialize-css.d.ts

```diff
@@ -468,4 +468,11 @@ interface JQuery {
      * @name method, the method to call (always "select_tab") and a param containing the id of the tab to open
      */
     tabs(method?:string, tab?:string): JQuery;
+    
+    /**
+     * <select> in material
+     * 
+     * @name method: callback
+     */
+    material_select(method?:string): JQuery;
```

Comment:
Wonderful, Thanks!

Just checked it to be sure:

```
material_select(method?:string): JQuery;
```
see first and last line of the selection: 
https://github.com/Dogfalo/materialize/blob/e0bce58c75bf0766ab692cfb9f3025a8c776ac09/js/forms.js#L299-L388

According to that it allows either the input 'destroy' or a callback function, which would end up as the following definition :
```
material_select(method?: string | () => void): JQuery;
```
(Typing this out of my head, but I think it's correct ts)

This is probably different from the other callbacks where this actually is a string, because it's an imported lib into MaterializeCSS I guess.

Could you maybe also add this to the test cases
```
$('select').material_select(() => console.log('callback'));
```
to materialize-css-tests.ts?
```
// Forms - Select

$('select').material_select();
$('select').material_select('destroy');
```